api/src/main/java/org/openmrs/module/dhisconnector/api/util/DHISConnectorUtil.java

```diff
@@ -46,7 +38,9 @@ public static String zipMappingBundle (String tempDirectory, List<File> fileList
                 while ((length = fis.read(bytes)) >= 0) {
                     zOut.write(bytes, 0, length);
                 }
+                fis.close();
```

Comment:
Do we close these if an exception is thrown?

Reply:
Yes, it is closed inside the `finally` block too. But had to keep it here too because it's inside a loop and can't wait until `finally` takes care of it after all the iterations.